x/ammswap/handler.go

```diff
@@ -295,7 +300,7 @@ func handleMsgTokenToNativeToken(ctx sdk.Context, k Keeper, msg types.MsgTokenTo
 	}
 	params := k.GetParams(ctx)
 	tokenBuy := calculateTokenToBuy(swapTokenPair, msg, params)
-	if tokenBuy.Amount.LT(msg.MinBoughtTokenAmount.Amount) {
+	if tokenBuy.IsZero() || tokenBuy.Amount.LT(msg.MinBoughtTokenAmount.Amount) {
 		return sdk.Result{
 			Code: sdk.CodeInternal,
 			Log:  fmt.Sprintf("Failed: expected minimum token to buy is %s but got %s", msg.MinBoughtTokenAmount, tokenBuy),
```

Comment:
It is suggested to return log like "selled token amount is too little to buy any token" when tokenBuy.IsZero() is true.